packages/app/src/app/pages/Dashboard/Components/NewTeamModal/TeamMembers.tsx

```diff
@@ -142,97 +142,132 @@ export const TeamMembers: React.FC<{
+    <>
       <Stack
-        as="form"
-        onSubmit={handleSubmit}
+        align="center"
         direction="vertical"
         gap={6}
-        css={{ width: '100%' }}
+        css={{
+          paddingTop: '60px',
+          paddingBottom: '40px',
+          maxWidth: '370px',
+          width: '100%',
+        }}
       >
-        <Textarea
-          label="Invite team members (Insert emails separated by a comma)"
-          name="members"
-          id="member"
-          autoFocus
-          required
-          rows={3}
-          value={addressesString}
-          onChange={e => {
-            setAddressesString(e.target.value);
+        <Text
+          as="h2"
+          size={32}
+          weight="500"
+          align="center"
+          css={{
+            margin: 0,
+            color: '#ffffff',
+            fontFamily: 'Everett, sans-serif',
+            lineHeight: '42px',
+            letterSpacing: '-0.01em',
           }}
-        />
-        {invalidEmails && invalidEmails.length > 0 ? (
-          <Text size={2} variant="danger">
-            {invalidEmails.length > 1
-              ? 'There seem to be errors in some email addresses, '
-              : 'Email is invalid, '}
-            please review: {formatInvalidEmails(invalidEmails)}
-          </Text>
-        ) : null}
+        >
+          {activeTeamInfo.name}
+        </Text>
+        <Stack
+          as="form"
+          onSubmit={handleSubmit}
+          direction="vertical"
+          gap={6}
+          css={{ width: '100%' }}
+        >
+          <Stack gap={4} direction="vertical">
+            <Textarea
+              id="member"
+              label="Invite team members (Insert emails separated by a comma)"
+              name="members"
+              value={addressesString}
+              onChange={e => {
+                setAddressesString(e.target.value);
+              }}
+              resize={false}
+              rows={3}
+              autoFocus
+              required
+            />
 
-        {inviteError ? (
-          <Text size={2} variant="danger">
-            {inviteError}
-          </Text>
-        ) : null}
+            {invalidEmails && invalidEmails.length > 0 ? (
+              <Text size={2} variant="danger">
+                {invalidEmails.length > 1
+                  ? 'There seem to be errors in some email addresses, '
+                  : 'Email is invalid, '}
+                please review: {formatInvalidEmails(invalidEmails)}
+              </Text>
+            ) : null}
 
-        <StyledButton loading={inviteLoading} type="submit">
-          Invite members
-        </StyledButton>
+            {inviteError ? (
+              <Text size={2} variant="danger">
+                {inviteError}
+              </Text>
+            ) : null}
+
+            <StyledButton loading={inviteLoading} type="submit">
+              Invite members
+            </StyledButton>
+          </Stack>
+          <Text
+            css={{
```

Comment:
We could add an `id` to this text and add `aria-describedby="text-id"` to the `TextArea` to associate the two if we want.

Reply:
@tristandubbeld nice! implemented here: [86e7e14](https://github.com/codesandbox/codesandbox-client/pull/7329/commits/86e7e1439b571b4813a3ce9af10e13637e1eb7d6)